### PR TITLE
Code formatting and changes

### PR DESCRIPTION
**Pull request by wattale**

### CODE REVIEW

File: components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtConfig.java [hunk 1]
```diff
@@ -549,7 +549,6 @@ private void loadPolicyExtensions(Properties properties, String extensionType) {
             className = properties.getProperty(extensionType + "." + count);
             if (className == null) {
                 count++;
-                size--;
```
Comment (johannnallathamby) [inline components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtConfig.java]: This looks a logic change ? Is this correct ?

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 2]
```diff
@@ -451,47 +452,47 @@ public IdentityProvider getResidentIdP(String tenantDomain)
...
-        if(IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
-                IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT) == null){
+        if (IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
+                IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT) == null) {
             Property sessionIdletimeOutProp = new Property();
             sessionIdletimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT);
-            String idleTimeout = IdentityUtil.getProperty("TimeConfig.SessionIdleTimeout");
-            if(StringUtils.isEmpty(idleTimeout)){
+            String idleTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.SESSION_IDLE_TIMEOUT);
+            if (StringUtils.isEmpty(idleTimeout)) {
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 3]
```diff
@@ -451,47 +452,47 @@ public IdentityProvider getResidentIdP(String tenantDomain)
...
-        if(IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
-                IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT) == null){
+        if (IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
+                IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT) == null) {
             Property rememberMeTimeOutProp = new Property();
             rememberMeTimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT);
-            String rememberMeTimeout = IdentityUtil.getProperty("TimeConfig.RememberMeTimeout");
-            if(StringUtils.isEmpty(rememberMeTimeout)){
+            String rememberMeTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.REMEMBER_ME_TIME_OUT);
+            if (StringUtils.isEmpty(rememberMeTimeout)) {
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 4]
```diff
@@ -451,47 +452,47 @@ public IdentityProvider getResidentIdP(String tenantDomain)
...
-        if(IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
-                IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD) == null){
+        if (IdentityApplicationManagementUtil.getProperty(sessionTimeoutConfig.getProperties(),
+                IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD) == null) {
             Property cleanUpPeriodProp = new Property();
             cleanUpPeriodProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD);
-            String cleanUpPeriod = IdentityUtil.getProperty("TimeConfig.PersistanceCleanUpPeriod");
-            if(StringUtils.isEmpty(cleanUpPeriod)){
+            String cleanUpPeriod = IdentityUtil.getProperty(IdentityConstants.ServerConfig.CLEAN_UP_PERIOD);
+            if (StringUtils.isEmpty(cleanUpPeriod)) {
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 5]
```diff
@@ -609,7 +610,7 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT) == null){
             Property rememberMeTimeOutProp = new Property();
             rememberMeTimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT);
-            String rememberMeTimeout = IdentityUtil.getProperty("TimeConfig.RememberMeTimeout");
+            String rememberMeTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.REMEMBER_ME_TIME_OUT);
             if(StringUtils.isEmpty(rememberMeTimeout)){
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 6]
```diff
@@ -595,7 +596,7 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT) == null){
             Property sessionIdletimeOutProp = new Property();
             sessionIdletimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT);
-            String idleTimeout = IdentityUtil.getProperty("TimeConfig.SessionIdleTimeout");
+            String idleTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.SESSION_IDLE_TIMEOUT);
             if(StringUtils.isEmpty(idleTimeout)){
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 7]
```diff
@@ -623,7 +624,7 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD) == null){
             Property cleanUpPeriodProp = new Property();
             cleanUpPeriodProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD);
-            String cleanUpPeriod = IdentityUtil.getProperty("TimeConfig.PersistanceCleanUpPeriod");
+            String cleanUpPeriod = IdentityUtil.getProperty(IdentityConstants.ServerConfig.CLEAN_UP_PERIOD);
             if(StringUtils.isEmpty(cleanUpPeriod)){
```
Comment (johannnallathamby) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Check StringUtils.isBlank()

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java [hunk 8]
```diff
@@ -54,18 +54,18 @@ public static AuthorizationGrantCache getInstance(int timeout) {
     }
 
     public void addToCache(CacheKey key, CacheEntry entry) {
-        String keyValue = ((AuthorizationGrantCacheKey)key).getUserAttributesId();
+        String keyValue = ((AuthorizationGrantCacheKey) key).getUserAttributesId();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java [hunk 9]
```diff
@@ -54,18 +54,18 @@ public static AuthorizationGrantCache getInstance(int timeout) {
...
-        if(enableRequestScopeCache){
-            SessionDataStore.getInstance().storeSessionData(keyValue,AUTHORIZATION_GRANT_CACHE_NAME,entry);
+        if (enableRequestScopeCache) {
+            SessionDataStore.getInstance().storeSessionData(keyValue, AUTHORIZATION_GRANT_CACHE_NAME, entry);
         }
     }
 
     public CacheEntry getValueFromCache(CacheKey key) {
-        String keyValue = ((AuthorizationGrantCacheKey)key).getUserAttributesId();
+        String keyValue = ((AuthorizationGrantCacheKey) key).getUserAttributesId();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java [hunk 10]
```diff
@@ -51,29 +51,29 @@ public static OAuthCache getInstance(int timeout) {
     }
 
     public void addToCache(CacheKey key, CacheEntry entry) {
-        String keyValue = ((OAuthCacheKey)key).getCacheKeyString();
+        String keyValue = ((OAuthCacheKey) key).getCacheKeyString();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java [hunk 11]
```diff
@@ -51,29 +51,29 @@ public static OAuthCache getInstance(int timeout) {
...
-        if(enableRequestScopeCache){
-            SessionDataStore.getInstance().storeSessionData(keyValue,OAUTH_CACHE_NAME,entry);
+        if (enableRequestScopeCache) {
+            SessionDataStore.getInstance().storeSessionData(keyValue, OAUTH_CACHE_NAME, entry);
         }
     }
 
     public CacheEntry getValueFromCache(CacheKey key) {
-        String keyValue = ((OAuthCacheKey)key).getCacheKeyString();
+        String keyValue = ((OAuthCacheKey) key).getCacheKeyString();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java [hunk 12]
```diff
@@ -51,29 +51,29 @@ public static OAuthCache getInstance(int timeout) {
...
-            cacheEntry = (CacheEntry) SessionDataStore.getInstance().getSessionData(keyValue,OAUTH_CACHE_NAME);
+        if (cacheEntry == null) {
+            cacheEntry = (CacheEntry) SessionDataStore.getInstance().getSessionData(keyValue, OAUTH_CACHE_NAME);
         }
         return cacheEntry;
     }
 
     public void clearCacheEntry(CacheKey key) {
-        String keyValue = ((OAuthCacheKey)key).getCacheKeyString();
+        String keyValue = ((OAuthCacheKey) key).getCacheKeyString();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java [hunk 13]
```diff
@@ -52,29 +52,29 @@ public static SessionDataCache getInstance(int timeout) {
     }
 
     public void addToCache(CacheKey key, CacheEntry entry) {
-        String keyValue = ((SessionDataCacheKey)key).getSessionDataId();
+        String keyValue = ((SessionDataCacheKey) key).getSessionDataId();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java [hunk 14]
```diff
@@ -52,29 +52,29 @@ public static SessionDataCache getInstance(int timeout) {
...
-            SessionDataStore.getInstance().storeSessionData(keyValue,SESSION_DATA_CACHE_NAME,entry);
+        SessionDataStore.getInstance().storeSessionData(keyValue, SESSION_DATA_CACHE_NAME, entry);
+        if (enableRequestScopeCache) {
+            SessionDataStore.getInstance().storeSessionData(keyValue, SESSION_DATA_CACHE_NAME, entry);
         }
     }
 
     public CacheEntry getValueFromCache(CacheKey key) {
-        String keyValue = ((SessionDataCacheKey)key).getSessionDataId();
+        String keyValue = ((SessionDataCacheKey) key).getSessionDataId();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java]: Wrong formatting. No need space.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java [hunk 15]
```diff
@@ -52,29 +52,29 @@ public static SessionDataCache getInstance(int timeout) {
...
-            cacheEntry = (CacheEntry) SessionDataStore.getInstance().getSessionData(keyValue,SESSION_DATA_CACHE_NAME);
+        if (cacheEntry == null) {
+            cacheEntry = (CacheEntry) SessionDataStore.getInstance().getSessionData(keyValue, SESSION_DATA_CACHE_NAME);
         }
         return cacheEntry;
     }
 
     public void clearCacheEntry(CacheKey key) {
-        String keyValue = ((SessionDataCacheKey)key).getSessionDataId();
+        String keyValue = ((SessionDataCacheKey) key).getSessionDataId();
```
Comment (johannnallathamby) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java]: Wrong formatting. No need space.

File: components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/user-mgt.jsp [hunk 16]
```diff
@@ -100,7 +101,7 @@
     java.lang.String errorAttribute = (java.lang.String) session.getAttribute(UserAdminUIConstants.DO_USER_LIST);
 
     String claimUri = request.getParameter("claimUri");
-    if (claimUri == null || claimUri.length() == 0) {
+    if (StringUtils.isEmpty(claimUri)) {
```
Comment (johannnallathamby) [inline components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/user-mgt.jsp]: Use StringUtils.isBlank()

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java [hunk 17]
```diff
@@ -84,10 +84,8 @@ public void handle(HttpServletRequest request, HttpServletResponse response) thr
                 // Retrieve AuthenticationRequestCache Entry which is stored stored from servlet.
                 if (sessionDataKey != null) {
                     log.debug("retrieving authentication request from cache..");
-                    AuthenticationRequestCacheKey cacheKey = new AuthenticationRequestCacheKey(
-                            sessionDataKey);
                     authRequest = (AuthenticationRequestCacheEntry) AuthenticationRequestCache
-                            .getInstance(0).getValueFromCache(cacheKey);
+                            .getInstance(0).getValueFromCache(sessionDataKey);
                 } else if (request.getParameter(FrameworkConstants.LOGOUT) == null ||
                            !"true".equals(request.getParameter(FrameworkConstants.LOGOUT))) {
```
Comment (johannnallathamby) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java]: Could avoid null check and equals check by using Booean.parseBoolean()

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java [hunk 18]
```diff
@@ -157,6 +157,10 @@ private WSTrust(){
             public static final String CLEAN_UP_TIMEOUT_DEFAULT = "20160";
             public static final String CLEAN_UP_PERIOD = "CleanUpPeriod";
             public static final String CLEAN_UP_PERIOD_DEFAULT = "1140";
+            public static final String TIME_CONFIG = "TimeConfig";
+            private IDPProperties(){
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: Formatting issues. Please fix.

Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: + We can possibly use a proper Runtime exception to indicate that it is not expected to instantiate IDPProperties even somewhere within IdentityApplicationConstants.

Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: You mean adding something like below ? 
private IDPProperties(){
    throw new RuntimeExceptions("Initiating IDPProperties is not allowed");
}

Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: Yeah, that's right. I was referring to something similar to what's mentioned below.

private IDPProperties () {
throw new AssertionError();
}

Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: I added the assertionError to all the inner static classes, with a message eg : 
private IDPProperties() {
      throw new AssertionError("Must not initiate an object of IDPProperties class");
}

Please review, Thanks.

Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java]: Looks good.

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 19]
```diff
@@ -569,8 +570,8 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT) == null){
             Property sessionIdletimeOutProp = new Property();
             sessionIdletimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.SESSION_IDLE_TIME_OUT);
-            String idleTimeout = IdentityUtil.getProperty("TimeConfig.SessionIdleTimeout");
-            if(StringUtils.isEmpty(idleTimeout)){
+            String idleTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.SESSION_IDLE_TIMEOUT);
+            if(StringUtils.isBlank(idleTimeout)){
```
Comment (prabathabey) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Formatting issues. Please fix.

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 20]
```diff
@@ -583,8 +584,8 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT) == null){
             Property rememberMeTimeOutProp = new Property();
             rememberMeTimeOutProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.REMEMBER_ME_TIME_OUT);
-            String rememberMeTimeout = IdentityUtil.getProperty("TimeConfig.RememberMeTimeout");
-            if(StringUtils.isEmpty(rememberMeTimeout)){
+            String rememberMeTimeout = IdentityUtil.getProperty(IdentityConstants.ServerConfig.REMEMBER_ME_TIME_OUT);
+            if(StringUtils.isBlank(rememberMeTimeout)){
```
Comment (prabathabey) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Formatting issues. Please fix.

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 21]
```diff
@@ -597,8 +598,8 @@ public void addResidentIdP(IdentityProvider identityProvider, String tenantDomai
                 IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD) == null){
             Property cleanUpPeriodProp = new Property();
             cleanUpPeriodProp.setName(IdentityApplicationConstants.Authenticator.IDPProperties.CLEAN_UP_PERIOD);
-            String cleanUpPeriod = IdentityUtil.getProperty("TimeConfig.PersistanceCleanUpPeriod");
-            if(StringUtils.isEmpty(cleanUpPeriod)){
+            String cleanUpPeriod = IdentityUtil.getProperty(IdentityConstants.ServerConfig.CLEAN_UP_PERIOD);
+            if(StringUtils.isBlank(cleanUpPeriod)){
```
Comment (prabathabey) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Formatting issues. Please fix.

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java (hunk 19 above)
Comment (wattale) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java (hunk 20 above)
Comment (wattale) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java (hunk 21 above)
Comment (wattale) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

**Pull request merged**